app/src/pages/explorer/dataset/DatasetGallery.vue

```diff
@@ -0,0 +1,271 @@
+        v-else-if="page > totalPages && totalPages > 0"
+      >
+        <h3 class="visualize_header-h3 u_margin-top-med">Invalid page number,
+          <a @click.prevent=loadPrevNextImage(totalPages)>return to page {{totalPages}}?</a>
+        </h3>
+      </div>
+      <div class="utility-roverflow u_centralize_text u_margin-top-med section_loader" v-else>
+        <h1 class="visualize_header-h1 u_margin-top-med">No Datasets Found</h1>
+      </div>
+    </div>
+    <dialogbox :active="dialogBoxActive" :minWidth="dialog.minWidth">
+      <template v-slot:title>{{dialog.title}}</template>
+      <template v-slot:content>
+        <div v-if="dialog.type=='delete'">
+          <md-content v-if="dialog.dataset">
+            <div> This will permanently remove the dataset <b>{{dialog.dataset.label}}</b> </div>
+            with identifier <b>{{dialog.dataset.identifier}}</b> and any associated files.
+          </md-content>
+        </div>
+        <div v-if="dialogLoading">
+          <spinner
+            :loading="dialogLoading"
+            text='Deleting Chart'
+          />
+        </div>
+      </template>
+      <template v-slot:actions>
+        <span v-if="dialog.type=='delete' && dialog.dataset">
+          <md-button @click.native.prevent="toggleDialogBox">
+            No, cancel
+          </md-button>
+          <md-button @click.native.prevent="deleteDataset(dialog.dataset)">
+            Yes, delete.
+          </md-button>
+        </span>
+        <md-button v-else @click.native.prevent="toggleDialogBox">Close</md-button>
+      </template>
+    </dialogbox>
+  </div>
+</template>
+<script>
+import spinner from '@/components/Spinner'
+import pagination from '@/components/explorer/Pagination'
+import Dialog from '@/components/Dialog.vue'
+import { mapGetters, mapActions, mapMutations } from 'vuex'
+import reducer from '@/mixins/reduce'
+import optionalChainingUtil from '@/mixins/optional-chaining-util'
+import explorerQueryParams from '@/mixins/explorerQueryParams'
+
+export default {
+  name: 'viz-grid',
+  mixins: [reducer, explorerQueryParams, optionalChainingUtil],
+  data () {
+    return {
+      loading: true,
+      pageNumber: 1,
+      baseUrl: `${window.location.origin}/api/files/`,
+      dialog: {
+        title: ''
+      },
+      dialogLoading: false,
+      filter: '',
+      searchWord: '',
+      searchEnabled: false
+    }
+  },
+  components: {
+    pagination,
+    spinner,
+    dialogbox: Dialog
+  },
+  computed: {
+    ...mapGetters({
+      dialogBoxActive: 'dialogBox',
+      isAuth: 'auth/isAuthenticated',
+      isAdmin: 'auth/isAdmin',
+      items: 'explorer/sddDatasets/getAllDatasets',
+      page: 'explorer/sddDatasets/getPage',
+      total: 'explorer/sddDatasets/getTotal',
+      totalPages: 'explorer/sddDatasets/getTotalPages',
+      queryTimeMillis: 'explorer/sddDatasets/getQueryTimeMillis'
+    })
+  },
+  methods: {
+    ...mapActions('explorer/sddDatasets', ['loadItems']),
```

Comment:
The mapAction ['loadItems'] is an unknown vuex action or mutation that doesn't exist and isn't called on the page